fedora/README.md

```diff
@@ -26,7 +26,7 @@ The metalink `http://mirrors.fedoraproject.org` is used to automatically select
 
 # Supported Docker versions
 
-This image is officially supported on Docker version 1.6.2.
+This image is officially supported on Docker version 1.6.2 and later.
```

Comment:
I'm just giving you a hard time. :heart:

I'm not opposed at all to adding this blurb in the template itself -- it
should be in "update.sh" or ".template-helpers" somewhere (can't remember
off-hand which has it).